packages/ckeditor5-code-block/src/converters.ts

```diff
@@ -174,10 +195,8 @@ export function dataViewToModelCodeBlockInsertion( editingView, languageDefs ) {
 		for ( const className of viewChildClasses ) {
 			const language = classesToLanguages[ className ];
 
-			if ( language ) {
-				writer.setAttribute( 'language', language, codeBlock );
-				break;
-			}
+			writer.setAttribute( 'language', language, codeBlock );
+			break;
```

Comment:
Why was this hanged? What about other classes that are not used for language?

Reply:
Good point, i;m not sure what happened here, reverting changes.